tests/integration_python/conftest.py

```diff
@@ -52,6 +52,11 @@ def test_data() -> Path:
     return Path(__file__).parents[1].joinpath("data").resolve()
 
 
+@pytest.fixture
+def import_files_dir() -> Path:
+    return repo_root() / "tests/data/import_files"
```

Comment:
This will only be used in `test_import`, so you might as well put the fixture there